code/game/objects/machinery/overwatch.dm

```diff
@@ -559,14 +597,16 @@ GLOBAL_LIST_EMPTY(active_cas_targets)
 		log_attack("[key_name(usr)] fired a [warhead_type]in for squad [current_squad] in [AREACOORD(T)].")
 		message_admins("[ADMIN_TPMONTY(usr)] fired a [warhead_type]for squad [current_squad] in [ADMIN_VERBOSEJMP(T)].")
 	visible_message(span_boldnotice("Orbital bombardment request accepted. Orbital cannons are now calibrating."))
-	send_to_squads("Initializing fire coordinates...")
+	send_to_squads("!!ORBITAL BOMBARDMENT INBOUND AT [get_area(selected_target)]!!")
```

Comment:
??

Reply:
This uses send order text now, avoids clutter & is clearer

Reply:
I meant why are you changing the wording
the extra !! dont really help for readability so im confused

Reply:
removed the !!

wording changes make it easier to understand (hopefully)